Setting apk directory is overwritten by automatic discovery

Probably this causes also 
```
Caused by: kotlin.KotlinNullPointerException
        at com.osacky.flank.gradle.FlankGradlePlugin.findDebugAndInstrumentationApk(FlankGradlePlugin.kt:95)
        at com.osacky.flank.gradle.FlankGradlePlugin.access$findDebugAndInstrumentationApk(FlankGradlePlugin.kt:12)
        at com.osacky.flank.gradle.FlankGradlePlugin$configureTasks$1.execute(FlankGradlePlugin.kt:30)
        at com.osacky.flank.gradle.FlankGradlePlugin$configureTasks$1.execute(FlankGradlePlugin.kt:12)
        at org.gradle.api.internal.plugins.DefaultPluginManager$3.execute(DefaultPluginManager.java:240)
        at org.gradle.api.internal.plugins.DefaultPluginManager$3.execute(DefaultPluginManager.java:238)
        at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:158)
        at org.gradle.api.internal.plugins.DefaultPluginManager.withPlugin(DefaultPluginManager.java:243)
        at com.osacky.flank.gradle.FlankGradlePlugin.configureTasks(FlankGradlePlugin.kt:28)
        at com.osacky.flank.gradle.FlankGradlePlugin.apply(FlankGradlePlugin.kt:17)
        at com.osacky.flank.gradle.FlankGradlePlugin.apply(FlankGradlePlugin.kt:12)
```

Maybe this should happen after evaluation?

--- a/buildSrc/src/main/java/com/osacky/flank/gradle/FlankGradlePlugin.kt
+++ b/buildSrc/src/main/java/com/osacky/flank/gradle/FlankGradlePlugin.kt
@@ -26,11 +26,6 @@ class FlankGradlePlugin : Plugin<Project> {
   }
 
   private fun configureTasks(project: Project, extension: FlankGradleExtension) {
-    project.pluginManager.withPlugin("com.android.application") {
-      if (extension.debugApk == null || extension.instrumentationApk == null) {
-        findDebugAndInstrumentationApk(project, extension)
-      }
-    }
     project.tasks.apply {
 
       register("downloadFlank", Download::class.java) {
@@ -42,6 +37,12 @@ class FlankGradlePlugin : Plugin<Project> {
     }
 
     project.afterEvaluate {
+      // Only use automatic apk path detection for 'com.android.application' projects.
+      project.pluginManager.withPlugin("com.android.application") {
+        if (extension.debugApk == null || extension.instrumentationApk == null) {
+          findDebugAndInstrumentationApk(project, extension)
+        }
+      }
       tasks.apply {
         createTasksForConfig(extension, extension, project, "")
 
@@ -93,7 +94,7 @@ class FlankGradlePlugin : Plugin<Project> {
   }
 
   private fun findDebugAndInstrumentationApk(project: Project, extension: FlankGradleExtension) {
-    val baseExtension = project.extensions.findByType(AppExtension::class.java)!!
+    val baseExtension = requireNotNull(project.extensions.findByType(AppExtension::class.java)) { "Could not find AppExtension in ${project.name}" }
     automaticallyConfigureTestOrchestrator(project, extension, baseExtension)
     baseExtension.applicationVariants.all {
       if (testVariant != null) {

--- a/sample/build.gradle
+++ b/sample/build.gradle
@@ -15,9 +15,9 @@ android {
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
-//    testOptions {
-//        execution 'ANDROIDX_TEST_ORCHESTRATOR'
-//    }
+    testOptions {
+        execution 'ANDROIDX_TEST_ORCHESTRATOR'
+    }
 }
 
 fladle {

